### PR TITLE
vr: restart conntrackd instead of '/usr/sbin/conntrackd -d'

### LINKED ISSUE #5138: conntrackd failed in VPC with Redundant VRs (4.15.1.0)
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
VPC VR
~~~

##### CLOUDSTACK VERSION

~~~
4.15.1.0-SNAPSHOT
~~~

##### CONFIGURATION

advanced network
vlan for isolation

##### OS / ENVIRONMENT

centos7 as hypervisors and mgt servers

##### SUMMARY

conntrackd failed in VPC with Redundant VRs (4.15.1.0)

##### STEPS TO REPRODUCE

~~~
1. create vpc with redundant vpc offering
2. create vpc tier
3. create vm in vpc tier
~~~

##### EXPECTED RESULTS

~~~
keepalived/conntrackd work.
~~~

##### ACTUAL RESULTS

~~~
keepalived works, but conntrackd failed.

```
root@r-94-VM:~# systemctl status conntrackd
● conntrackd.service - Conntrack Daemon
   Loaded: loaded (/lib/systemd/system/conntrackd.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Tue 2021-06-22 09:59:38 UTC; 24min ago
     Docs: man:conntrackd(8)
           man:conntrackd.conf(5)
  Process: 2010 ExecStart=/usr/sbin/conntrackd -C /etc/conntrackd/conntrackd.conf (code=exited, status=1/FAILURE)
 Main PID: 2010 (code=exited, status=1/FAILURE)

Jun 22 09:59:38 r-94-VM systemd[1]: conntrackd.service: Main process exited, code=exited, status=1/FAILURE
Jun 22 09:59:38 r-94-VM systemd[1]: conntrackd.service: Failed with result 'exit-code'.
Jun 22 09:59:38 r-94-VM systemd[1]: Failed to start Conntrack Daemon.
Jun 22 09:59:38 r-94-VM systemd[1]: conntrackd.service: Service RestartSec=100ms expired, scheduling restart.
Jun 22 09:59:38 r-94-VM systemd[1]: conntrackd.service: Scheduled restart job, restart counter is at 5.
Jun 22 09:59:38 r-94-VM systemd[1]: Stopped Conntrack Daemon.
Jun 22 09:59:38 r-94-VM systemd[1]: conntrackd.service: Start request repeated too quickly.
Jun 22 09:59:38 r-94-VM systemd[1]: conntrackd.service: Failed with result 'exit-code'.
Jun 22 09:59:38 r-94-VM systemd[1]: Failed to start Conntrack Daemon.
```
~~~

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR fixes #5138 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [x] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan package

**Review by rhtyd — APPROVED** [review]

File: systemvm/debian/opt/cloud/bin/cs/CsRedundant.py [hunk 1]
```diff
@@ -280,7 +280,7 @@ def set_backup(self):
             dev = interface.get_device()
 
         cmd = "%s -C %s" % (self.CONNTRACKD_BIN, self.CONNTRACKD_CONF)
```
Comment (rhtyd) [inline systemvm/debian/opt/cloud/bin/cs/CsRedundant.py:282]: @weizhouapache if `cmd` is unused can we remove it?

Comment (weizhouapache) [reply systemvm/debian/opt/cloud/bin/cs/CsRedundant.py:282]: @rhtyd sure...

**Comment by weizhouapache** [comment]
@blueorangutan test

File: systemvm/debian/opt/cloud/bin/cs/CsRedundant.py [hunk 2]
```diff
@@ -242,7 +242,6 @@ def set_fault(self):
         for interface in interfaces:
             CsHelper.execute("ifconfig %s down" % interface.get_device())
 
-        cmd = "%s -C %s" % (self.CONNTRACKD_BIN, self.CONNTRACKD_CONF)
         CsHelper.execute("%s -s" % cmd)
```
Comment (rhtyd) [inline systemvm/debian/opt/cloud/bin/cs/CsRedundant.py:245]: @weizhouapache wrong `cmd` was removed?

**Comment by rhtyd** [comment]
@weizhouapache everytime any code is changed, pl kick bo package before kicking testing

File: systemvm/debian/opt/cloud/bin/cs/CsRedundant.py (hunk 2 above)
Comment (weizhouapache) [reply systemvm/debian/opt/cloud/bin/cs/CsRedundant.py]: @rhtyd 
sorry, fixed now

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan test

**Review by rhtyd — APPROVED** [review]

**Comment by rhtyd** [comment]
@blueorangutan test

**Review by nvazquez — APPROVED** [review]
LGTM

**Pull request merged**